codes/classical/properties/random.yml

```diff
@@ -27,7 +27,7 @@ realizations:
     - 'Distributed storage systems \cite{doi:10.1109/TPDS.2008.251}.'
 
 notes:
-    - 'Shannon''s pioneering work \cite{doi:10.1002/j.1538-7305.1948.tb01338.x} analyzes the distance distribution of the code given a rate. Given \(N\) and the rate \(R\), the minimum distance of a TRC is given by the GV bound \(d=N\delta_{GV}(2R)\), where \(\delta_{GV} = h^{-1}(1-R)\), \(0\le R \le 1\), and \(\delta_{GV}(x)=0\) for all other \(R\). For a TLC, the minimum distane is given by \(d=N\delta_{GV}(R)\).'
+    - 'Shannon''s pioneering work \cite{doi:10.1002/j.1538-7305.1948.tb01338.x} analyzes the distance distribution of the code given a rate. Given \(N\) and the rate \(R\), the minimum distance of a TRC is given by the GV bound \(d=N\delta_{GV}(2R)\), where \(\delta_{GV} = h^{-1}(1-R)\), \(0\le R \le 1\), and \(\delta_{GV}(x)=0\) for all other \(R\). For a TLC, the minimum distance is given by \(d=N\delta_{GV}(R)\).'
```

Comment:
Thanks. The other two comments I addressed in the other commit. Please resubmit those.